managed/ui/src/components/common/associatedUniverse/AssociatedUniverse.scss

```diff
@@ -0,0 +1,8 @@
+.associated-universe-status {
+  margin: 0;
```

Comment:
Is it required??

Reply:
Yes, the Default universe-status class has margin and line-height set to some value which will not fit here.